operator/src/main/java/oracle/kubernetes/operator/calls/unprocessable/Cause.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.calls.unprocessable;
+
+public class Cause {
+  private String reason;
+  private String message;
```

Comment:
how would message or field ever get set?

Reply:
In this case, it wouldn't. This is one of a set of classes which map to the content of the ApiException's 422 status. So far, I have not found a use for those fields, but since they are in the structure, I mapped them.

Reply:
ok